.github/workflows/main-build.yml

```diff
@@ -13,18 +13,14 @@ jobs:
 
     steps:
     - uses: actions/checkout@v3
-
-    - name: Fetch viash
-      run: |
-        bin/init
-        bin/viash -h
-        bin/viash --version
-
+    - uses: viash-io/viash-actions/setup@v1.0.0
     - name: Fetch viash tools
-      run: |
-
-        bin/init_tools ${{ secrets.GTHB_USER }} ${{ secrets.GTHB_PAT }}
-        tree bin/tools
+      uses: actions/checkout@v3
+      with:
+        repository: "viash-io/viash_tools"
+        token: ${{ secrets.GTHB_PAT }}
+        ref: "main_build"
+        path: "viash_tools"
```

Comment:
Can we clone this to `$HOME/viash_tools` or `../viash_tools` so it doesn't accidentally get committed?

Reply:
I don't think you can use `../viash_tools`. I think checkout does not work outside `$GITHUB_WORKSPACE`.

Reply:
I am creating actions for generating schemas and parameter files, tough. So lets come back to this PR once those are created. https://github.com/viash-io/viash-actions/pull/2

Reply:
Oops, ok!

Reply:
Action for generating schemas and parameter files added!